lib/bigstringaf.mli

```diff
@@ -268,3 +268,9 @@ val unsafe_blit_to_bytes : t -> src_off:int -> Bytes.t -> dst_off:int -> len:int
 
 val unsafe_memcmp        : t -> int -> t      -> int -> int -> int
 val unsafe_memcmp_string : t -> int -> string -> int -> int -> int
+
+
+(** {2 IO } *)
```

Comment:
Adding UNIX system call support in the `bigstringaf` library is a non-starter. Ss @dinosaure correctly pointed out, that would severely hinder portability of a library that is a the bottom of a the dependency graph for a lot of portably libraries. 

This could be included in a `bigstringaf-unix` library, if that's something you're interested in doing.

Reply:
removed the additions for now.